modules/appeals_api/app/services/appeals_api/supporting_evidence/evidence_uploader.rb

```diff
@@ -15,7 +17,11 @@ def initialize(appeal, document, type:)
       def process!
         generate_evidence_submission!
         uploader.store!(document)
-        update_submission!
+        update_submission!('submitted')
+      rescue => e
+        log_message_to_sentry('Error saving to S3', :warning, error: e.to_s)
```

Comment:
Let's make sure we're also sending the GUID of the submission and the exception message to sentry. Maybe the supportable_id and type too?